src/Param.h

```diff
@@ -22,6 +22,20 @@ struct DomainSize
 	double height_;
 };
 
+struct InitialInfections
+{
+  int number_[MAX_NUM_SEED_LOCATIONS];
+  int do_random_loc_;
+  int do_all_in_same_loc_;
+  int min_pop_dens_;
```

Comment:
Same as above, don't abbreviate. `min_population_density`

I'm okay with using abbreviations for things that have a smaller scope, but for member variables that are used throughout, I prefer to make it clearer to the reader.

Reply:
I'd maybe even go so far as even expand `min` to `minimum`...

But if you disagree, I'd at LEAST like it to be `min_pop_density` :)

Reply:
Done